### PR TITLE
Add detail to SOPS / Azure guide

### PR DESCRIPTION
**Pull request by apeschel**
Provide a more in-depth guide on how to set the environment variables
required by SOPS to access an Azure Key Vault.

### CODE REVIEW

File: docs/guides/mozilla-sops.md [hunk 1]
```diff
@@ -141,6 +141,55 @@ When using Azure Key Vault you need to authenticate the kustomize controller eit
...
+  --from-literal=AZURE_TENANT_ID="XXX" \
+  --from-literal=AZURE_TENANT_ID="XXX" \
+  --from-literal=AZURE_TENANT_ID="XXX"
+```
+
+You'll need a separate process from Flux for bootstrapping this specific secret
+before you bootstrap Flux, or you'll end up with a dependency cycle.
+
+Finally, update your kustomize controller deployment definition in
+`flux-system/gotk-components.yaml` to mount the secret data as environment
```
Comment (stefanprodan) [inline docs/guides/mozilla-sops.md:177]: Please see the docs on how to do this without altering the bootstrap files https://toolkit.fluxcd.io/guides/installation/#customize-flux-manifests. Manual edits to the components manifests means that users can no longer upgrade Flux automatically.

**Review by stefanprodan — CHANGES_REQUESTED** [review]

**Comment by stealthybox** [comment]
@apeschel I'm sorry I didn't see this patch before, or I would have worked with you on it.

I just PR'd some overlapping docs on SOPS + Flux in #1064

File: docs/guides/mozilla-sops.md (hunk 1 above)
Comment (apeschel) [reply docs/guides/mozilla-sops.md:177]: @stefanprodan Thanks, this makes a lot of sense! I was wondering if there was a way to do this, but hadn't found the guide.

**Comment by apeschel** [comment]
@stealthybox Your PR looks good, I'd be willing to coordinate with you still, if you're willing. I should have some time in the next few days.

**Comment by stefanprodan** [comment]
Closing this is in favour of #1064

**Pull request closed without merging**